orderapi/src/main/java/fi/hel/verkkokauppa/order/api/admin/OrderAdminController.java

```diff
@@ -30,9 +30,7 @@ public class OrderAdminController {
     @GetMapping(value = "/order-admin/get", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<OrderAggregateDto> getOrder(@RequestParam(value = "orderId") String orderId) {
         try {
-            Order order = orderService.findById(orderId);
```

Comment:
Removed this unnecessary order fetch -  `orderService.orderAggregateDto(...)` is performing the fetching of order data.

Reply:
Good find! This kind of refactoring I like a lot!